test/fixtures/moby-runc.yml

```diff
@@ -36,8 +51,12 @@ targets: # Distro specific build requirements
       runtime:
         /bin/sh:
         libseccomp:
-          - ">= 2.3"
+          version:
+            - ">= 2.3"
     tests:
```

Comment:
should `jammy` target also have some tests?

Reply:
I moved the manpage test from mariner into the main tests.
I can't think of any jammy specific ones to add here.